tests/reftests/pin.test

```diff
@@ -1623,3 +1623,35 @@ Done.
 file1
 ### opam-cat OPAM/flag-edit/.opam-switch/overlay/tide/opam | grep install
 install: [["mkdir" "%{lib}%/%{name}%"] ["cp" "file1" "%{lib}%/%{name}%/file1"]]
+### :C:h: opam pin doesn't upgrade outdated dependencies
+### opam switch create no-upgrade --empty
+### <pkg:dep.1>
+opam-version: "2.0"
+### <pkg:dep.2>
+opam-version: "2.0"
+### <pkg:pkg.1>
+opam-version: "2.0"
```

Comment:
From description, i have the feeling that there is a missing `depends: "dep"`, no ?